### PR TITLE
Various minor UX cleanup

### PR DESCRIPTION
**Pull request by jgerigmeyer**
## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

- https://trello.com/c/MQiUZB8a
- https://trello.com/c/dBZcwuYM
- https://trello.com/c/ugTzOpj7

## Description
_The commit messages say what you did; this should explain why and/or how._
- [x] Description is in Trello card

- Use "Add task" consistently instead of "Create task"
- Show help text for disabled Org Type radios (in edit-task modal, if task has an org) in form instead of tooltip.
- Default sort projects in list (review, in-progress, planned, merged)

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- project-detail
- task-detail, edit-task modal
- repo-detail (project-list)

## Show me
_Provide screenshots/animated gifs/videos if necessary._

Screenshot 2020-07-02 11 42 51

Screenshot 2020-07-02 11 43 10

Screenshot 2020-07-02 11 41 54

### CODE REVIEW

**Comment by jgerigmeyer** [comment]
[Review "Create a Task" vs "Add a Task" copy for consistency.](https://trello.com/c/MQiUZB8a/198-review-create-a-task-vs-add-a-task-copy-for-consistency)

**Comment by jgerigmeyer** [comment]
[Move disabled org-type help text from Tooltip into body text.](https://trello.com/c/dBZcwuYM/197-move-disabled-org-type-help-text-from-tooltip-into-body-text)

**Comment by jgerigmeyer** [comment]
[Default sort projects by status.](https://trello.com/c/ugTzOpj7/196-default-sort-projects-by-status)

File: src/js/components/projects/table/index.tsx [hunk 1]
```diff
@@ -26,9 +28,18 @@ const ProjectTable = ({
...
   }));
+  const statusOrder = {
+    [PROJECT_STATUSES.REVIEW]: 1,
+    [PROJECT_STATUSES.IN_PROGRESS]: 2,
+    [PROJECT_STATUSES.PLANNED]: 3,
+    [PROJECT_STATUSES.MERGED]: 4,
+  };
+  const projectDefaultSort = sortBy(items, [
+    (item) => statusOrder[item.status],
+  ]);
```
Comment (jgerigmeyer) [inline src/js/components/projects/table/index.tsx:39]: @wlonk I did this sort on the front-end, but then realized that (unlike tasks) projects are paginated... so for this to work, I think we actually want to apply this to the default model `ordering` on the back end -- so it happens pre-pagination. Is that something we can do? Ideally, sort by `status` (review, then in-progress, then planned, then merged), then by `-created_at`, finally by `name`.

Comment (wlonk) [reply src/js/components/projects/table/index.tsx:39]: That is absolutely something we can do. Do we want it to be a default ordering applied in all cases unless specifically overridden? If so, we just add it to the `Meta` on that model.

Comment (wlonk) [reply src/js/components/projects/table/index.tsx:39]: Commit to relevance-order in.

File: metecho/api/models.py [hunk 2]
```diff
@@ -575,7 +575,11 @@ def finalize_available_task_org_config_names(self, originating_user_id=None):
         self.notify_changed(originating_user_id=originating_user_id)
 
     class Meta:
-        ordering = ("-created_at", "name")
+        ordering = (
+            "status",
+            "-created_at",
+            "name",
+        )
```
Comment (jgerigmeyer) [inline metecho/api/models.py:582]: @wlonk 

1. This is missing a migration, but...
2. Is this valuable to have at all, given the real sorting is now happening in the serializer? That is, we don't actually want to sort by "status" as it does here, so what's the point?

File: metecho/api/views.py [hunk 3]
```diff
@@ -191,6 +199,18 @@ class ProjectViewSet(CreatePrMixin, ModelViewSet):
...
+        qs = super().get_queryset()
+        whens = [
+            When(status=PROJECT_STATUSES.Review, then=0),
+            When(status=PROJECT_STATUSES["In progress"], then=1),
+            When(status=PROJECT_STATUSES.Planned, then=2),
+            When(status=PROJECT_STATUSES.Merged, then=3),
+        ]
+        return qs.annotate(ordering=Case(*whens, output_field=IntegerField())).order_by(
+            "ordering"
+        )
```
Comment (jgerigmeyer) [inline metecho/api/views.py:212]: @wlonk If `ordering` is the same between two models, it looks like it's no longer applying the secondary sort of `-created_at`.

File: metecho/api/models.py (hunk 2 above)
Comment (wlonk) [reply metecho/api/models.py:582]: Oops, left that in when I didn't mean to, anyway.

**Comment by jgerigmeyer** [comment]
I think this is ready for merge once @stacyk signs off 👍

**Comment by jgerigmeyer** [comment]
> Everything that I was able to see looked good. I couldn't get anything in the review stage though (project nor task in order to see project review status). When I tried to create a test org, I kept getting an error. Maybe I have too many created? So I deleted a dev org and then got the same error when I tried to create the test org again. I'm not sure if it is something with my local or something I need to report.

@stacyk The most likely culprit is creating an org from an old (outdated) branch. First thing I'd try (other than a Docker rebuild) is to create a new project, new task, then try again. (I just confirmed that new orgs are creating okay for me, so it's not likely a code bug.)

**Pull request merged**